dbux-graph-host/src/graph/controllers/HiddenNodeManager.js

```diff
@@ -17,8 +17,11 @@ export default class HiddenNodeManager extends HostComponentEndpoint {
       const visible = this.shouldBeVisible(runNode);
       this._setVisible(runNode, visible);
     }
-    this._notifyStateChanged();
     this._notifyHiddenCountChanged();
+
+    this.waitForUpdate()
+      .then(() => this._notifyStateChanged())
+      .catch(err => this.logger.error(err));
```

Comment:
Contextualize the error, so we understand what happened from the log message.
E.g.: `this.logger.error(`notifyStateChanged failed -`, err)`